actions/qemuimg2disk/v1/README.md

```diff
@@ -0,0 +1,23 @@
+---
+slug: qemuimg2disk
+name: qemuimg2disk
+tags: disk
+maintainers: Nahum Shalman <nahum.shalman@gmail.com>
+description: "This action will stream a remote disk image to a block device, and
+is mainly used to write cloud images to a disk.
```

Comment:
I think there is something wrong around here maybe a missed double quote

Reply:
fixed